html2asketch/page.js

```diff
@@ -1,9 +1,8 @@
 import Base from './base';
 
 class Page extends Base {
-
-  constructor({width, height}) {
-    super();
+  constructor({width, height, id = 'Page'}) {
+    super({id});
```

Comment:
I think it's the same as with the Document - these ids are not used right now, and since theoretically there can be more that one page, it makes more sense to generate random values instead of hardcoding 'Page' for everything.